pkg/loader/compose/compose.go

```diff
@@ -363,6 +363,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		err1 := checkLabelsPorts(len(serviceConfig.Port), composeServiceConfig.Labels["kompose.service.type"], name)
+		if err1 != nil {
+			return kobject.KomposeObject{}, errors.Wrap(err, err1.Error())
```

Comment:
what's going on here? for errors.Wrap it should be `errors.Wrap(err, "Your error message here") or something along the lines.